### PR TITLE
Use real colors for the color buttons on the LED strip configuration tab

### PR DESCRIPTION
**Pull request by joefarmer**
This is a pull request for issue #95

### CODE REVIEW

File: tabs/led_strip.css [hunk 1]
```diff
@@ -144,7 +144,7 @@
     margin: 3px 0;
 }
 
-.tab-led-strip button:hover {
+.tab-led-strip .not(colors) button:hover {
```
Comment (thenickdude) [inline tabs/led_strip.css]: :not(.colors) surely?

Comment (joefarmer) [inline tabs/led_strip.css]: Oh wow. Yes, it should have been. I've updated it so it doesn't need it and is black-text friendly.

**Comment by hydra** [comment]
The only problem here is that the colors wont match if the user has changed the defaults but we can change that later.

**Comment by hydra** [comment]
The PR was made against the wrong branch, merged to development.  Please review 3d822e836e09a694e3a04707c8d859259ee426a6

**Pull request closed without merging**